src/component/Text.tsx

```diff
@@ -240,7 +240,7 @@ export const Text = ({
   if (scaleToFit) {
     const lineWidth = wordsByLines[0].width;
     const { width } = props;
-    transforms.push(`scale(${(isNumber(width as number) ? (width as number) / lineWidth : 1) / lineWidth})`);
+    transforms.push(`scale(${isNumber(width as number) ? (width as number) / lineWidth : 1})`);
```

Comment:
We should probably have a unit test in place for this in general even if it can't be perfect. Not a blocker for this for now, but if we can add it we should to prevent regressions

Reply:
I think it would be easier to add this, if it's actually fixed? I guess we could check that it never gets too small now, but if the actual behavior is in place, I imagine, you could test for that?

Reply:
this is good for now though